app/src/main/java/com/pixlee/pixleeandroidsdk/SampleActivity.java

```diff
@@ -176,7 +176,9 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
      */
     private void createAlbum() {
         Context c = this.getApplicationContext();
+
         PXLClient.initialize("196i8ZzIAhKU8dO2kDe");
```

Comment:
dumb question, you can fire this function without the 2nd param?

Reply:
yes, I have 2 constructors in PXLClient